test/python/test_gravity_model.py

```diff
@@ -171,3 +171,39 @@ def test_polyhedral_evaluable_pickle(
     acceleration = np.array([result[1] for result in sol])
     np.testing.assert_array_almost_equal(potential, expected_potential)
     np.testing.assert_array_almost_equal(acceleration, expected_acceleration)
+
+
+def test_polyhedron_metric() -> None:
+    """Tests the metric conversion/ options
+    """
+    computation_point = np.array([1.0, 0.0, 1.0])
+    gravity_constant = 6.67430e-11
+    meter_polyhedron = Polyhedron(
+        polyhedral_source=(CUBE_VERTICES, CUBE_FACES),
+        density=1.0,
```

Comment:
same as above, testing for another density may be good?

Reply:
Absolutley agreed!